website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -329,10 +329,16 @@ The name and tag of the image (`IMG=<some-registry>/<project-name>:tag`) in both
 
 #### Deploy the operator
 
-For this example we will run the operator in the `default` namespace which can be specified for all resources in `config/default/kustomization.yaml`:
+Per default a new namespace is created with name `<project-name>-system`. For this example we will run the operator in the `memcached-operator` namespace which can be specified for all resources in `config/default/kustomization.yaml`:
 
 ```sh
-$ cd config/default/ && kustomize edit set namespace "default" && cd ../..
+$ cd config/default/ && kustomize edit set namespace "memcached-operator" && cd ../..
+```
+
+For convenience we will save it for all subsequent kubectl commands:
+
+```sh
+$ kubectl config set-context --current --namespace=memcached-operator
```

Comment:
I think the above step is not required.

Reply:
It is not strictly required but commands that follow in the tutorial would need to have the namespace specified:
$ kubectl get pods
$ kubectl get deployment
...

Reply:
IHMO: the goal of the tutorial is to describe how to develop the operator and how it works. In this way, IMO use the -namespace=name in the commands make more intuitive what is happening.